src/widget/wmainmenubar.cpp

```diff
@@ -599,6 +607,23 @@ void WMainMenuBar::initialize() {
     addMenu(pHelpMenu);
 }
 
+WMainMenuBar::~WMainMenuBar() {
+    qDebug() << "Delete WMainMenuBar";
+
+    delete m_bShowMenuBar;
```

Comment:
make this a shared or unique ptr so we don't have to do manual memory management

Reply:
Done.